plugins/UsersManager/UsersManager.php

```diff
@@ -152,7 +153,7 @@ public static function isValidPasswordString($input)
         return $l >= self::PASSWORD_MIN_LENGTH;
     }
 
-    public static function checkPassword($password)
+    public static function checkPassword($password, $showError = false)
```

Comment:
A type hint on the new parameter would be nice along with a PHPDoc header for the method, also maybe `returnError` would be a better name since the method doesn't output anything itself?